cmd/meroxa/root/apps/init.go

```diff
@@ -151,7 +151,8 @@ func (i *Init) Execute(ctx context.Context) error {
 	}
 
 	i.logger.Infof(ctx, "Turbine Data Application successfully initialized!\n"+
-		"You can start interacting with Meroxa in your app located at \"%s/%s\"", i.path, name)
+		"You can start interacting with Meroxa in your app located at \"%s/%s\".\n"+
+		"Your Application will not be visible in the Meroxa Dashboard until after deployment.", i.path, name)
```

Comment:
@janelletavares I thought about including a message here regarding `meroxa apps describe %s --extend` since that's being the most useful command to run after deploying. What do you think?

Reply:
@janelletavares oops! my bad. I was thinking about `apps deploy`, you're right!